### ISSUE TITLE
Write-Progress and Cmdlet.WriteProgress() display 0% complete as 100% complete

### ISSUE THREAD

**Issue by twest820** [PowerShell/PowerShell#15524]
## Steps to reproduce
```powershell
for ($remaining = 3; $remaining -gt 0; $remaining -= 3)
{
    Write-Progress -Activity "Demonstrating 0% complete = 100% complete..." -PercentComplete 0 -SecondsRemaining $remaining
    Start-Sleep -Seconds $remaining
}
```

## Expected behavior
An empty progress bar (black) is displayed as is done by PowerShell 7.1 and earlier. (I'm also a bit surprised this wasn't caught by automated UX tests.)

## Actual behavior
The progress bar is yellow all the way across. The black portion of the bar is correctly sized for 1% or more complete in the cases I've spot checked.

## Environment data
```none
PSVersion                      7.2.0-preview.6
PSEdition                      Core
GitCommitId                    7.2.0-preview.6
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by ajansveld** [comment]
Is this on the roadmap for 7.3?

**Comment by EmeraldFlame** [comment]
I am able to replicate here on 7.2.1 stable. It is a fairly large annoyance.

**Comment by iSazonov** [comment]
I'd expect it is very simple fix.

**Comment by chenxizhang** [comment]
same issue here. now I have to set the PercentComplete  to 1 to avoid the problem.

**Comment by zett42** [comment]
Still reproducible as of PS 7.3.1

**Comment by dwtaber** [comment]
I can reproduce in 7.3.3, but not in 7.4.0-preview.2.  I haven't spotted a relevant pull request yet, but it looks like this was addressed at some point.

**Comment by ajansveld** [comment]
This was fixed in https://github.com/PowerShell/PowerShell/pull/18735 (as stated in https://github.com/PowerShell/PowerShell/issues/18850, which is a duplicate of this issue).
This issue can be closed.

**Comment by twest820** [comment]
Still repros for me in 7.3.9. Closing therefore seems premature at best. The linked PRs went in months ago and therefore have had plenty of time to propagate. But they somehow haven't.

Not seeing there's any release date for PowerShell 7.4.0 where a fix could be confirmed and, given the .NET 8.0 tie, if there is in fact a fix it won't be practically observable for some time.

**Comment by theJasonHelmick** [comment]
Thank you @twest820 - The Interactive Working group has reviewed this and this doesn't repo in 7.4. We believe this was recently fixed, so may still repo in 7.3.x.
